src/index.ts

```diff
@@ -6,3 +6,4 @@ export * from './lambda'
 export * from './number'
 export * from './object'
 export * from './string'
+export * from './phone'
```

Comment:
A gente tinha comentado com a galera de deixar a exportação dos arquivos na mesma ordem das pastas, que é em ordem alfabética, então só ajustei isso 😄 
```suggestion
export * from './phone'
export * from './string'
```